src/components/Description/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import styles from "./index.module.css";
+
+const Description = ({ children, theme, variant }) => {
```

Comment:
This might seem a bit stupid.... can we name it `BigText` ? :joy:

Reply:
😅 I will change it with the promise to change it later for a better name for thins thing. 
#textBetweenTitleAndSubtitle #subsubtitle